mkdocs.yml

```diff
@@ -0,0 +1,10 @@
+site_name: avalanche-cli
+site_description: |
+  Avalanche-CLI is a command-line interface for building, deploying, and maintaining Avalanche
+  Subnets. It can be used to prototype and test Subnets, as well as to deploy them to the
```

Comment:
would prefer the inverse order, or make it very clear that it is a very good option to deploy subnets to avalanche (it seems from the paragraph that its main goal is prototype and testing? and secondary, deploy)

Reply:
rewrote